src/main/java/Duke.java

```diff
@@ -1,10 +1,167 @@
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.stage.Stage;
+import java.io.FileNotFoundException;
+import java.time.format.DateTimeParseException;
+
 public class Duke {
```

Comment:
since there are no other classes that extends from Duke, you could make it a `final` class.